physics/test/FiniteElementFluxes_test.cpp

```diff
@@ -221,12 +221,12 @@ TEST_CASE("Freezing conditions", "[FiniteElementFluxes]")
     public:
         AtmosphereData()
         {
-            registerProtectedArray(ProtectedArray::T_AIR, &tair);
-            registerProtectedArray(ProtectedArray::DEW_2M, &tdew);
-            registerProtectedArray(ProtectedArray::P_AIR, &pair);
-            registerProtectedArray(ProtectedArray::WIND_SPEED, &windSpeed);
-            registerProtectedArray(ProtectedArray::SW_IN, &sw_in);
-            registerProtectedArray(ProtectedArray::LW_IN, &lw_in);
+            getStore().registerArray(Protected::T_AIR, &tair);
```

Comment:
Yeah, definitely would prioritise that test class clear up if possible (or do as you go along --- probably easier)